src/js/api/ChatAPI.js

```diff
@@ -23,7 +23,20 @@ var ChatAPI = {
     },
     submitRoom : function(submitRoom){
```

Comment:
I always find this name akward. I'd rather read `createRoom`. Even though Firebase *might* fail creating the room, I believe the naming should not reflect that and be more pro-active.